x-pack/platform/plugins/shared/stack_connectors/server/connector_types/inference/inference.ts

```diff
@@ -269,7 +269,7 @@ export class InferenceConnector extends SubActionConnector<Config, Secrets> {
       false,
       signal
     );
-    return response.rerank!;
+    return response.rerank!.map(({ relevance_score: score, ...rest }) => ({ score, ...rest }));
```

Comment:
Rather than renaming this back to `score` we should change all of the places where it is used to be `relevance_score`
I've created a follow up issue to cover this work.
https://github.com/elastic/kibana/issues/216263